apple/Tests/TestCases/Profile/AppPreferencesTests.swift

```diff
@@ -17,8 +17,26 @@ final class AppPreferencesTests: Test<AppPreferences> {
 	}
 	
 	func test_has_gateway() {
-		XCTAssertTrue(SUT.default.hasGateway(with: .init(string: "https://mainnet.radixdlt.com/")!))
-		XCTAssertTrue(SUT.default.hasGateway(with: .init(string: "https://mainnet.radixdlt.com")!))
-		XCTAssertFalse(SUT.default.hasGateway(with: .init(string: "https://radixdlt.com")!))
+		// Valid URLs
+		do {
```

Comment:
for expected NON-throws I prefer changing the test method to be throwing, and remove the `do {} catch {}`.